tests/unit_tests/test_db_handler.py

```diff
@@ -76,6 +76,22 @@ def test_reading_db_sst(db):
         assert pars["camera_pixels"] == 2048
 
 
+def test_get_reference_data(db):
+
+    logger.info("----Testing reading reference data-----")
+    pars = db.getReferenceData("south", "Current")
```

Comment:
Is it by design that this test should fail when the `Current` model changes?

Reply:
No, for the others it is but in this case it I changed it to Prod5.